preferences.json

```diff
@@ -3,10 +3,10 @@
     "windowWidth" : 740.0,
     "windowHeight" : 600.0,
     "windowCoordinates" : {
-      "x" : 350,
-      "y" : 83
+      "x" : 214,
+      "y" : 36
     }
   },
-  "patientRecordFilePath" : "data/patientRecord.json",
-  "drugInventoryFilePath" : "data/drugInventory.json"
-}
+  "patientRecordFilePath" : "data\\patientRecord.json",
+  "drugInventoryFilePath" : "data\\drugInventory.json"
+}
```

Comment:
perhaps we can consider ignoring the preferences file, I believe it keeps alternating between "/  and "\\" across Mac for the former and Windows for the latter.